package.json

```diff
@@ -59,7 +59,7 @@
   "dependencies": {
     "busboy": "0.2.14",
     "hawk": "7.0.7",
-    "request": "2.85.0",
+    "request": "2.86.0",
```

Comment:
Shouldn't this have an updated package-lock.json as well?

Reply:
⬇️ Added it.